server/controllers/userControllers.js

```diff
@@ -0,0 +1,27 @@
+import User from '../models/user'
+export default new class userControllers {
+    async save(req, res){
```

Comment:
Great work. Anyway, is there a validation to check if the user you're trying to create already exists?